packages/manager/modules/web-universe-components/src/order/order.component.js

```diff
@@ -0,0 +1,154 @@
+    */
+    catalog: '<',
+
+    /* 'catalogItemTypeName':
+      Item type of which we want catalog informations, as the catalog contains
+      'addons' and 'plans' where product informations can be found.
+
+      Check CATALOG_ITEM_TYPE_NAMES constant.
+    */
+    catalogItemTypeName: '<',
+
+    /* 'productName':
+      The product name to use, to add items for item order.
+    */
+    productName: '<',
+
+    /* 'sendCurrentState':
+      Method to send component current state, so the upper scope is aware of
+      the loading state of the component.
+
+      Example :
+      In controller :
+
+      getOrderState(state) { this.orderState = state; }
+
+      ...later:
+      if (this.orderState.isLoading) { // Some code }
+    */
+    sendCurrentState: '&',
+
+    /* 'serviceNameToAddProduct' :
+      Service name of which we will add product/addon.
+      If set, the order will consist to add option to an existing service.
+      If null, the order concerns an new product.
+    */
+    serviceNameToAddProduct: '<?',
+
+    /* 'type':
+      Type of order, see ORDER_TYPES constant.
+    */
+    type: '<',
+
+    /* 'user':
+      User informations, for cart creation, price format style.
+    */
+    user: '<',
+
+    /* 'onBeforePricingGetPlancode':
+      Method called to get the product/addon plan code, which will be called
+      after configuration phase, and before pricing step.
+      Plan code example : webHosting, cloudDB.
+    */
+    onBeforePricingGetPlancode: '&',
+
+    /* 'onCheckoutSuccess':
+      Method called once the order is checked out.
+    */
+    onCheckoutSuccess: '&',
+
+    /* 'onError':
+      Error handler method.
+    */
+    onError: '&',
+
+    /* 'onGetConfiguration':
+      Method to get configuration items that will be added to the order
+      cart, called when fetching checkout informations.
+
+      The configuration key list must follow catalog product
+      required configuration.
+      This can be found through, GET /order/catalog/public/{productName},
+      in plans > planItem > configurations.
+
+      Example for cloudDB (GET /order/catalog/public/cloudDB,
+      with FR subsidiary):
+      configurations = [
+        {
+          isCustom: false,
+          values: [ "gra1" ],
+          name: "dc",
+          isMandatory: true
+        },
+        {
+          name: "engine"
+          values: [
+            "redis_3.2"
+            "redis_4.0"
+            ...
+          ],
+          isCustom: false,
+          isMandatory: true
+        }
+      ]
+      Here, the configuration label will be the 'name' property value,
+      and the configuration value, one of the 'values' list.
+
+      Returned value must be of the following format:
+      [
+        {
+          label: configurationLabel (ex: 'legacy_domain'),
+          value: configurationValue (ex: 'www.ovhcloud.com'),
+        }
+      ];
+    */
+    onGetConfiguration: '&',
```

Comment:
I know that it is common practice to prefix callback by `on` but it sounds quite odd
```suggestion
    getConfiguration: '&',
```
? maybe instead

Reply:
No, it will be triggered when you get the configuration, it won't help you get the configuration